src/ros_main.cpp

```diff
@@ -109,6 +110,7 @@ int main(int argc, char **argv)
 
   ROSController *controller(nullptr);
   ActionServer *action_server(nullptr);
+  URScriptHandler *urscript_handler(nullptr);
```

Comment:
This should probably not be a pointer, `controller` and `action_server` are pointers here because they're exclusive of each other.
(You also forgot to delete the pointer although that doesn't really matter 😄)

Reply:
Sure. I thought I keep the same approach, but no problem to make it local variable :)